docs/learn/migrate/migrate-erc721-to-lsp8.md

```diff
@@ -130,19 +209,103 @@ const name = ethers.toUtf8String(nameValue);
+
+For dApp developers, you can import the data keys from the `@lukso/lsp-smart-contracts` and use them directly in your scripts via _ethers.js_ or _web3.js_.
+
+```javascript
+import { ERC725YDataKeys } from '@lukso/lsp-smart-contracts';
+
+const nameKey = ERC725YDataKeys.LSP4.LSP4TokenName;
+const symbolKey = ERC725YDataKeys.LSP4.LSP4TokenSymbol;
+
+const nameValue = await token.getData(nameKey); // 0x4c5350382050726f66696c65
+const symbolValue = await token.getData(symbolKey); // 0x4c5350382050726f66696c65
+
+const name = ethers.toUtf8String(nameValue); // Cool NFT
+const symbol = ethers.toUtf8String(symbolValue); // COOL
+```
+
+</TabItem>
+
+<TabItem value="erc725.js" label="erc725.js" attributes={{className: "tab_erc725"}}>
+
+You can also obtain the [full schema definition](../../tools/erc725js/schemas.md) of the LSP4 Metadata from [`@erc725/erc725.js`](../../tools/erc725js/getting-started.md) library. **This library will also help you to [encode easily](../../tools/erc725js/methods.md#encodedata) data key value pairs.**
+
+```javascript
+import { ERC725 } from '@erc725/erc725.js';
+import LSP4Schema from '@erc725/erc725.js/schemas/LSP4DigitalAssetMetadata.json';
+
+const nameKey = LSP4Schema.find((schema) => schema.name == 'LSP4TokenName').key;
+const symbolKey = LSP4Schema.find(
+  (schema) => schema.name == 'LSP4TokenSymbol',
+).key;
+
+const nameValue = await token.getData(nameKey); // 0x4c5350382050726f66696c65
+const symbolValue = await token.getData(symbolKey); // 0x4c5350382050726f66696c65
+
+const [name, symbol] = ERC725.decodeData([
+  {
+    keyName: 'LSP4TokenName',
+    value: nameValue,
+  },
+  {
+    keyName: 'LSP4TokenSymbol',
+    value: symbolValue,
+  },
+]);
+/**
+[
+  {
+    name: 'LSP4TokenName',
+    key: '0xdeba1e292f8ba88238e10ab3c7f88bd4be4fac56cad5194b6ecceaf653468af1',
+    value: "Cool NFT",
+  },
+  {
+    name: 'LSP4TokenSymbol',
+    key: '0x2f0a68ab07768e01943a599e73362a0e17a63a72e94dd2e384d2c1d4db932756',
+    value: "COOL",
+  },
+]
+*/
+```
+
+</TabItem>
+
+</Tabs>
+
 ### Extended Token Metadata
 
-:::info
+:::success Tutorial 🎥
 
-To learn more about setting and creating the LSP4Metadata JSON, check the [metadata](../digital-assets/metadata-management/metadata-preparation.md) section.
+See the [**Metadata Management**](../digital-assets/metadata-management/edit-token-metadata.md) guide + video for how to create and set the JSON metadata for your LSP8 Token.
 
 :::
 
-LSP8 allows for more flexible and extensive metadata storage directly on-chain. You can store a JSON object containing information about the whole NFT contract and for each individual tokenId.
+LSP8 allows for more flexible and extensible metadata. You can store a JSON object containing information about the whole NFT contract and for each individual tokenId. The [`LSP4Metadata`](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#lsp4metadata) is a JSON object that can contain many information about the token, including:
```

Comment:
```suggestion
LSP8 allows for more flexible and extensible metadata. You can store a JSON object containing information about:
- the whole NFT Collection contract 
- and for each individual NFT `tokenId`. 

The [`LSP4Metadata`](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#lsp4metadata) is a JSON object that can contain many information, including:
```

Reply:
We should also add a quick link to this page: https://docs.lukso.tech/standards/tokens/LSP8-Identifiable-Digital-Asset#lsp8-collection-vs-tokenid-metadata